xbmc/music/MusicDatabase.cpp

```diff
@@ -3380,6 +3383,7 @@ bool CMusicDatabase::GetArtistsByWhere(const std::string& strBaseDir, const Filt
 
         pItem->GetMusicInfoTag()->SetDatabaseId(artist.idArtist, MediaTypeArtist);
         pItem->SetIconImage("DefaultArtist.png");
+        pItem->GetMusicInfoTag()->SetArtist(artist);
```

Comment:
Is this necessary? Shouldn't this be covered by passing the `CArtist` object to `CFileItem`'s constructor?